pantos/common/configuration.py

```diff
@@ -179,15 +184,25 @@ def __parse_file(self, path: pathlib.Path) -> dict[str, typing.Any]:
             _logger.info('loading env variables from environment defined file '
                          'PANTOS_ENV_FILE')
             env_files.insert(0, pathlib.Path(os.environ['PANTOS_ENV_FILE']))
+
+            # Extend env_files with .env paths from _CONFIGURATION_PATHS
+        env_files.extend(
+            pathlib.Path(str(p)).with_name(
+                str(p.name) + self.default_file_name + '.env')
+            for p in _CONFIGURATION_PATHS)
         # Iterate over the potential .env file paths
         for env_file in env_files:
-            if env_file.is_file():
-                try:
+            try:
+                if env_file.is_file():
                     dotenv.load_dotenv(env_file)
                     _logger.info(f'loaded .env from file {env_file}')
                     break
-                except Exception:
-                    raise ConfigError(f'unable to load .env file {env_file}')
+            except OSError as error:
+                # Perhaps the path is not readable
+                _logger.warning(f'error while reading: {env_file}', error)
+                pass
+            except Exception:
```

Comment:
Do we need to throw an error here? Or should it just be a warning?

Reply:
It's not fatal at this point, is it? I think a warning is more appropriate in this case.